src/pre_defined_classes/mathematical_program/hard/q3/MainTest.java

```diff
@@ -0,0 +1,27 @@
+package pre_defined_classes.mathematical_program.hard.q3;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    //parsons
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The area of the sector is: "),
+                new DoubleLiteral("math")
+        };
+    }
+
+    public void runMain() {
+        AreaSector.main(new String[]{});
+    }
+
+    @Test
+    public void mathTest() {
+        assertEquals(24.801128670839425, Double.parseDouble(getItemByName("math")), 0.1, "Your program's math calculations are incorrect");
+    }
+}
```

Comment:
EOF newline (this means you're missing a new line at the end of the file, this is good practice for using git version controlling and just in general tbh)

Reply:
Do we want an extra empty line at the end of file? Also how does this relate to git version control?